exercises/triangle/canonical-data.json

```diff
@@ -8,88 +8,115 @@
+        "your language."
     ],
-    "cases": [
+    "equilateral": {
+      "description": "returns true if the triangle is equilateral",
+      "cases": [
         {
-            "description": "equilateral triangle has all sides equal",
-            "sides": [2, 2, 2],
-            "expected": "equilateral"
+              "description": "true if all sides are equal",
+              "sides": [2, 2, 2],
+              "expected": true
         },
         {
-            "description": "larger equilateral triangle",
-            "sides": [10, 10, 10],
-            "expected": "equilateral"
+              "description": "false if any side is unequal",
+              "sides": [2, 3, 2],
+              "expected": false
         },
         {
-            "description": "isosceles triangle with last two sides equal",
-            "sides": [3, 4, 4],
-            "expected": "isosceles"
+              "description": "false if no sides are equal",
+              "sides": [5, 4, 6],
+              "expected": false
         },
         {
-            "description": "isosceles triangle with first two sides equal",
-            "sides": [4, 4, 3],
-            "expected": "isosceles"
+              "description": "All zero sides are illegal, so the triangle is not equilateral",
+              "sides": [0, 0, 0],
+              "expected": false
         },
         {
-            "description": "isosceles triangle with first and last sides equal",
-            "sides": [4, 3, 4],
-            "expected": "isosceles"
+              "#": "Your track may choose to skip this test and deal only with integers if appropriate",
+              "description": "sides may be floats",
+              "sides": [0.5, 0.4, 0.6],
+              "expected": true
+        }
+      ]
+    },
+    "isosceles": {
+      "description": "returns true if the triangle is isosceles",
+      "cases": [
+        {
+              "description": "true if last two sides are equal",
+              "sides": [3, 4, 4],
+              "expected": true
         },
         {
-            "description": "isosceles triangle with unequal side larger than equal sides",
-            "sides": [4, 7, 4],
-            "expected": "isosceles"
+              "description": "true if first two sides are equal",
+              "sides": [4, 4, 3],
+              "expected": true
         },
         {
-            "description": "scalene triangle has no equal sides",
-            "sides": [3, 4, 5],
-            "expected": "scalene"
+              "description": "true if first and last sides are equal",
+              "sides": [4, 3, 4],
+              "expected": true
         },
         {
-            "description": "2a == b+c looks like equilateral, but isn't always",
-            "sides": [5, 4, 6],
-            "expected": "scalene"
+              "description": "true if three sides are equal",
+              "sides": [4, 4, 4],
+              "expected": true
```

Comment:
Might be good to add a comment or expansion of the description here that clarifies that this is BOTH isosceles and equilateral.